app/helpers/container_helper/textual_summary.rb

```diff
@@ -4,12 +4,12 @@ module ContainerHelper::TextualSummary
   #
 
   def textual_group_properties
-    items = %w(image name state restart_count backing_ref image_ref)
+    items = %w(name state restart_count backing_ref)
     items.collect { |m| send("textual_#{m}") }.flatten.compact
   end
 
   def textual_group_relationships
-    items = %w(ems container_replicator container_group container_project)
+    items = %w(ems container_replicator container_group container_project container_image)
```

Comment:
> The idea is to go from the most generic to the most specific. So basically in an ems you have multiple container_projects, each project may have container_replicators that are pointing to container_groups that are using container_images.

pods are not using images, containers are.
But regardless, I have a dilemma re the order on summary pages. 
I'm thinking about ordering them by how "close" or relevant they are to the entity on which summary's page they currently reside. Because when looking at containers for example, images and pods are "level 1", while replicators are more further and less interesting. User reads top down, so most relevant information should be read first, hence my dilemma :)
@serenamarie12 - any thoughts on this topic?
(the general solution/fixing is out of scope for this patch, but it would be nice to get to some guideline to follow on all pages)